docs/source-pytorch/index.rst

```diff
@@ -251,6 +251,7 @@ Current Lightning Users
    Pretrained models <advanced/pretrained>
    Profiler <tuning/profiler>
    Pruning and Quantization <advanced/pruning_quantization>
+   Post-training Quantization <advanced/post_training_quantization>
```

Comment:
Since we already have a quantization page in the menu, this link is redundant.

Reply:
@edenlightning The 'post_training_quantization' page is newly added there is a need to expose the link from up-level user document per previous comments. Any suggestions on how to make this page easily accessible by users?